site/en/codelabs/openthread-network-simulator/index.lab.md

```diff
@@ -30,14 +30,8 @@ adapt to topology changes to avoid single point of failure.
 
 > aside positive
 >
-> **Dive Deeper:** Refer to
-[Thread Overview](https://www.threadgroup.org/What-is-Thread/Overview)
-for more on Thread.
-
-[OpenThread](https://openthread.io/) released by Google is an open-source
```

Comment:
@jwhui Do we want to keep this, but just remove the version number?  So it would just say [Thread Specification].  I don't know if "support all features in the specification" is accurate or not.

If not, maybe we just keep the first sentence.

Reply:
I would say keep the sentence and just remove the version number.